src/main/scala/ai/privado/passes/SQLPropertyPass.scala

```diff
@@ -0,0 +1,100 @@
+package ai.privado.passes
+
+import ai.privado.cache.RuleCache
+import ai.privado.languageEngine.java.language.NodeStarters
+import ai.privado.model.sql.{SQLColumn, SQLQuery}
+import ai.privado.tagger.PrivadoParallelCpgPass
+import ai.privado.utility.SQLParser
+import io.shiftleft.codepropertygraph.generated.{Cpg, EdgeTypes}
+import io.shiftleft.codepropertygraph.generated.nodes.{
+  File,
+  JavaProperty,
+  NewFile,
+  NewSqlColumnNode,
+  NewSqlQueryNode,
+  NewSqlTableNode
+}
+import org.slf4j.LoggerFactory
+
+class SQLPropertyPass(cpg: Cpg, projectRoot: String, ruleCache: RuleCache)
+    extends PrivadoParallelCpgPass[JavaProperty](cpg) {
+
+  val logger = LoggerFactory.getLogger(getClass)
+
+  override def generateParts(): Array[_ <: AnyRef] =
+    cpg.property.l.filter(prop => prop.name.matches("(?i)query")).toArray
+
+  override def runOnPart(builder: DiffGraphBuilder, property: JavaProperty): Unit = {
+    addSqlNodes(builder, property)
+  }
+
+  private def addSqlNodes(builder: SQLPropertyPass.this.DiffGraphBuilder, property: JavaProperty): Unit = {
+
+    val query        = property.value
+    val lineNumber   = property.lineNumber.getOrElse(-1).asInstanceOf[Int]
```

Comment:
Instead of assingning -1 as reading time, better when we create Nodes in Property pass, we assign the lineNumber as -1 and columnNumber as -1, if we are unable to fetch the value

Reply:
Property Node has `Option[None]` for line and column by default. Do we still need to store it?